interfaces/HO-Portal/src/app/auth/auth.service.ts

```diff
@@ -71,11 +71,6 @@ export class AuthService {
         if (user.role) {
           user.roles = [user.role];
         }
-
-        // 'Clean' roles-object into flat list of roles
```

Comment:
@elwinschmitz this code is not needed any more I think with the back-end change below. You need to remove this same piece of code also from the AW-app PR.